.github/workflows/aws-ci.yml

```diff
@@ -0,0 +1,47 @@
+
+name: AWS CI
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+      - dev
+      - 'feature/**'
+
+permissions:
+  id-token: write
+
+jobs:
+  run-ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
```

Comment:
Sorry I missed it on the first pass, but can we pin this to a commit? I generally prefer to in case things break, but especially for the credentials handling.

Reply:
Fixed